### ISSUE TITLE
Shortix Backup Wiped my SD Card

### ISSUE THREAD

**Issue by Pleasance13** [Jannomag/shortix#11]
I pointed the Shortix backup to my SD Card during installation and I guess because I didn't specify "sd-root/shortix" as the location, it fucking overwrote everything on there.

**Comment by Jannomag** [comment]
That's bad...
I need to figure out why. I already found an issue when the script doesn't use kdialog.
Did you have dialog windows or did the installer ask you its questions through the terminal?

**Comment by Pleasance13** [comment]
Yeah. And I'm sorry for the bluntness, it was just a lot of data loss for me.

I had dialog windows.

**Comment by Jannomag** [comment]
I tried Shortix after removing it completely from my system.
For this I created a directory with random files and folder in it and chose this directory on kdialog. 
It worked without deleting any files - and it can't do this actually. It just uses the `cp` command. So I can't reproduce your issue.
I just noticed that it didn't copy all files and symlinks, just the script files, which I was already able to fix.

I don't have an sd card in my deck but since it's mounted to usual directory there shouldn't be any differences to what I did for testing.

So you just used the dialog to chose the path and also created a new directory on your desired path? (I did exactly this on my test).

Edit:
Found the issue. It can happen that the script delete everything if you didn't create a subdirectory for Shortix backup. I will fix this. I'm very sorry for your data loss!
